### PR TITLE
Added new intent UI for sell intent

### LINKED ISSUE #60151: Seller Experience: Add a feature flag.
Let's add a feature flag to Calypso for the Seller Experience project – maybe just `seller-experience`.

### LINKED ISSUE #60153: Seller Experience: Add sellers to the existing intent capture screen.
We need to add sellers to the existing intent capture screen. As with all work until launch, it will need to be behind our feature flag (#60151 ).

Screen Shot 2022-01-17 at 11 53 39 AM

### LINKED ISSUE #60154: Seller Experience: Add the store name and tagline view.
The seller intent flow requires an optional store name and tagline view.

Screen Shot 2022-01-17 at 11 54 12 AM

### LINKED ISSUE #60155: Seller Experience: Offer users a simple or WooCommerce-based choice.
Our MVP will give users two choices for the kind of store they want to create: a simple store (based on Payments blocks), or a full-featured option (based on WooCommerce).

Screen Shot 2022-01-17 at 11 54 30 AM

### PR DESCRIPTION
**Pull request by nelsonec87**
#### Changes proposed in this Pull Request

* Added new "Sell" intent to the `/start/` flow under the `seller-experience` feature flag.

image

image

image

#### Testing instructions

* Access `start/setup-site/intent?siteSlug=<SITE SLUG>` (which doesn't enable the `seller-experience` feature flag)
* Things should be unchanged!

* Access `start/setup-site/intent?siteSlug=<SITE SLUG>&flags=seller-experience` (which enables the `seller-experience` feature flag)
* Check the "Sell" intent
* Click on it
* Check Store details form
* Click Continue
* Check the new options (Simple, More Power)

Related to #60151
Related to #60153
Related to #60155
Related to #60154

### CODE REVIEW

File: client/signup/steps/intent/intent-screen.tsx [hunk 1]
```diff
@@ -33,6 +33,14 @@ const useIntents = ( { translate }: Pick< Props, 'translate' > ): Intent[] => {
 			value: 'build',
 			actionText: translate( 'Start building' ),
 		},
+		{
+			key: 'sell',
+			title: translate( 'Sell' ),
```
Comment (a8ci18n) [inline client/signup/steps/intent/intent-screen.tsx:41]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **15** times:
`translate( 'sell', 'block search term' )` *ES Score: 10*

File: client/signup/steps/intent/intent-screen.tsx [hunk 2]
```diff
@@ -33,6 +33,14 @@ const useIntents = ( { translate }: Pick< Props, 'translate' > ): Intent[] => {
...
+			description: translate( 'Sell your products' ),
```
Comment (a8ci18n) [inline client/signup/steps/intent/intent-screen.tsx:42]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **3** times:
`translate( 'Sell your products online' )` *ES Score: 7*

File: client/signup/steps/sell-details/sell-details.tsx [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+	const handleSubmit = ( event: React.FormEvent ) => {
+		event.preventDefault();
+		onSubmit( formValues );
+	};
+
+	return (
+		<form className="sell-details__form" onSubmit={ handleSubmit }>
+			<FormFieldset className="sell-details__form-fieldset">
+				<FormLabel htmlFor="siteTitle" optional>
+					{ translate( 'Store name' ) }
```
Comment (a8ci18n) [inline client/signup/steps/sell-details/sell-details.tsx:47]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **24** times:
`translate( 'Store Name' )` *ES Score: 9*

File: client/signup/steps/sell-details/sell-details.tsx [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+				/>
+			</FormFieldset>
+			<FormFieldset className="sell-details__form-fieldset">
+				<FormLabel htmlFor="tagline" optional>
+					{ translate( 'Tagline' ) }
+				</FormLabel>
+				<FormInput name="tagline" id="tagline" value={ formValues.tagline } onChange={ onChange } />
+				<FormSettingExplanation>
+					<Icon className="sell-details__form-icon" icon={ tip } size={ 20 } />
+					{ translate( 'In a few words, explain what your store is about.' ) }
```
Comment (a8ci18n) [inline client/signup/steps/sell-details/sell-details.tsx:63]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **49** times:
`translate( 'In a few words, explain what your blog is about.' )` *ES Score: 7*

**Comment by nelsonec87** [comment]
No longer needed

**Pull request closed without merging**